stdlib/public/core/VarArgs.swift

```diff
@@ -290,7 +290,9 @@ final public class VaListBuilder {
     // supported vararg type is greater than the alignment of Int.
     // FIXME: this implementation is not portable because
     // alignof differs from the ABI alignment on some architectures
-#if os(watchOS) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
+    // FIXME: This is valid for AAPCS calling convention, FreeBSD/arm
+    // may need this too.
+#if (os(watchOS) || os(Linux)) && arch(arm)   // FIXME: rdar://21203036 should be arch(armv7k)
```

Comment:
@jrose-apple I'm about to push this fix, but I want to ask whether you want to keep the fixme (referencing the rdar) about armv7k.  It doesn't seem like a v7k issue.